components/FormModal.js

```diff
@@ -3,15 +3,22 @@ import Form from "@/components/Form";
 import SendingConfirmation from "@/components/SendingConfirmation";
 import { mutate } from "swr";
 import { useEffect } from "react";
+import { useSession } from "next-auth/react";
 
 export default function FormModal({ showModal }) {
+  const session = useSession();
+
   async function handleCreate(formData) {
     const url = "/api/games";
 
+    const isLoggedIn = session.status === "authenticated";
+    const user = session.data.user;
+    const userCreatedGame = { ...formData, user };
```

Comment:
For security reasons, it is better to add the user in the backend just before adding the element to the database. This ensures that nobody can manipulate the information that is added into the database.